### PR TITLE
[Feature:TAGrading] Remove right arrow for annotation

### PR DESCRIPTION
**Pull request by micpap25**
Currently there is a right arrow on the TAGrading page which allows users to go to the annotation area for PDFs and view other files in their own full panel. However, this functionality is not useful for most files (which can just be popped out instead) and for PDFs there is already a way to view them and annotate (clicking the PDF's name). This removes the redundant functionality.

In accordance with #10618

Before:
image

After:
image

### CODE REVIEW

**Comment by williamjallen** [comment]
Can you please add before and after screenshots?

File: site/app/templates/grading/electronic/SubmissionPanel.twig [hunk 1]
```diff
@@ -86,11 +86,10 @@
     {% if not (blind_grader and (dir == 'queue_file.json' or dir == '.user_assignment_access.json')) %}
     <div>
         <div class="file-viewer">
-            <a class='openAllFile{{ title }} openable-element-{{ title }} key_to_click' file-url="{{ path|url_encode }}" onclick='openFrame("{{ dir|url_encode|e('js') }}", "{{ path|url_encode|e('js') }}", "{{ id }}"); updateCookies();' data-viewer_id="{{ id }}">
+            <a data-testid='open-file' oclass='openAllFile{{ title }} openable-element-{{ title }} key_to_click' file-url="{{ path|url_encode }}" onclick='openFrame("{{ dir|url_encode|e('js') }}", "{{ path|url_encode|e('js') }}", "{{ id }}"); updateCookies();' data-viewer_id="{{ id }}">
```
Comment (williamjallen) [inline site/app/templates/grading/electronic/SubmissionPanel.twig:89]: Was this deliberate, or just a typo?

```suggestion
            <a data-testid='open-file' class='openAllFile{{ title }} openable-element-{{ title }} key_to_click' file-url="{{ path|url_encode }}" onclick='openFrame("{{ dir|url_encode|e('js') }}", "{{ path|url_encode|e('js') }}", "{{ id }}"); updateCookies();' data-viewer_id="{{ id }}">
```

Comment (micpap25) [reply site/app/templates/grading/electronic/SubmissionPanel.twig]: Typo!! Apologies

**Comment by micpap25** [comment]
@bmcutler What should we do with this? Based on our conversations I feel this could either be merged or closed, I just want your opinion.

**Comment by bmcutler** [comment]
I don't think there is a hurry to merge this minor change.  
So we will close it now and wait until a more substantial and impactful UI design update is proposed and implemented.

**Pull request closed without merging**